fitbenchmarking/results_output.py

```diff
@@ -18,21 +22,28 @@
 FILENAME_EXT_TXT = 'txt'
 FILENAME_EXT_HTML = 'html'
 
+HTML_COLUR_SCALE = ['#fef0d9', '#fdcc8a', '#fc8d59', '#e34a33', '#b30000']
```

Comment:
Typo in COLOUR (or COLOR if you were going for the spelling that was there before)